templates/silicon/index.html

```diff
@@ -0,0 +1,550 @@
+        </a>
+      </p>
+    </div>
+    <div class="p-testimonial__author">
+      <p class="u-no-margin--bottom u-no-padding--top" style="font-weight: 500;">Manuvir Das</p>
+      <p class="u-text--muted">
+        Vice President of Enterprise Computing at NVIDIA
+      </p>
+    </div>
+  </div>
+  <hr class="is-fixed-width" />
+  <div class="p-testimonial">
+    <div class="p-testimonial__logo u-pad-logo">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/f8ef46d3-AMD_Xilinx_logo.svg",
+        alt="AMD Xilinx logo",
+        width="96",
+        height="60",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="p-testimonial__quote">
+      <p class="u-no-padding--top">
+        <i>
+          &ldquo;Xilinx has been on a journey to simplify embedded design while empowering developers with the capability of adaptive computing. Developers that are familiar with C++, Python, ROS 2, or their preferred machine-learning framework on Ubuntu should be able to access Xilinx benefits without traditional FPGA development easily, and this is what Canonical and Xilinx are accelerating together.&rdquo;
+        </i>
+      </p>
+      <p>
+        <a href="https://ubuntu.com/blog/canonical-teams-with-xilinx-to-accelerate-the-development-of-adaptive-socs">
+          Read the blog
+        </a>
+      </p>
+    </div>
+    <div class="p-testimonial__author">
+      <p class="u-no-margin--bottom u-no-padding--top" style="font-weight: 500;">Chetan Khona</p>
+      <p class="u-text--muted">
+        Senior Director for Industrial, Vision, Healthcare and Sciences at AMD-Xilinx
+      </p>
+    </div>
+  </div>
+  <hr class="is-fixed-width" />
+  <div class="p-testimonial">
+    <div class="p-testimonial__logo u-pad-logo">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/1f876369-advantech.svg",
+        alt="Advantech logo",
+        width="150",
+        height="30",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="p-testimonial__quote">
+      <p class="u-no-padding--top">
+        <i>
+          &ldquo;We are very pleased with our collaboration with Canonical and the expansion of our operating system offerings. This collaboration will enable us to satisfy even more customer requirements and deliver an integrated, pre-validated, and flexible open-computing gateway platform that allows fast solution development and deployment.&rdquo;
+        </i>
+      </p>
+      <p>
+        <a href="https://ubuntu.com/blog/advantech-and-canonical-form-strategic-partnership-for-iot-gateways">
+          Read the blog
+        </a>
+      </p>
+    </div>
+    <div class="p-testimonial__author">
+      <p class="u-no-margin--bottom u-no-padding--top" style="font-weight: 500;">Miller Chang</p>
+      <p class="u-text--muted">
+        Vice president of Advantech Embedded Computing Group
+      </p>
+    </div>
+  </div>
+</section>
+
```

Comment:
The the copy doc there is a CTA for 'find a partner' and 'become a partner', it is in one of the designs but not the other. Have we chosen to not include this?

Reply:
I have added it anyway, I'd like to think @lyubomir-popov maybe have just omitted it.